tool/bindings_generator/translator.dart

```diff
@@ -31,11 +33,14 @@ class _Library {
     final name = named.name.toDart;
     assert(!translator._typeToLibrary.containsKey(name));
     translator._typeToLibrary[named.name.toDart] = this;
+    translator._typeToDeclaration[named.name.toDart] = node;
```

Comment:
(not for this CL): just realized webidl_api could be updated to start using primitive types directly. It will be a small but nice cleanup on the generator

Reply:
Yup, I'll make a note to address this in another CL.